requirements.txt

```
@@ -1,10 +1,10 @@
-hepunits==1.1.*
-particle==0.9.*
-decaylanguage==0.6.*;python_version>="3.4"
-awkward==0.12.*
+hepunits~=1.2.0
+particle~=0.11.0
+decaylanguage~=0.7.0
+awkward~=0.12
 uproot-methods==0.7.*
```

```suggestion
uproot-methods~=0.7.0
```
To match style with the rest of the PR can also switch here.